crates/bin/pcli/src/command/tx.rs

```diff
@@ -1280,6 +1335,105 @@ impl TxCmd {
             TxCmd::Position(PositionCmd::Replicate(replicate_cmd)) => {
                 replicate_cmd.exec(app).await?;
             }
+            TxCmd::DutchAuctionSchedule {
+                source,
+                amount,
+                recieve: _,
+                auction_duration,
+                starting_price,
+                reserve_price,
+                fee_tier,
```

Comment:
The goal of these commands is to be able to exercise the DA logic, IMO it would be clearer to have them take the exact fields that an auction description specifies i.e.

- a start height
- an end height
- a step count
- an input value
- a minimum output
- a maximum output

ditto for the rest of the commands. We are the customers for this command so being able to easily specify custom descriptions is a feature

Reply:
ah good call, I was thinking of the customer in the context of the end user rather than us. It makes sense since we're still in dev mode. I'll make the necessary changes

Reply:
modified fields to match design doc